lib/src/test/kotlin/starknet/data/TypedDataTest.kt

```diff
@@ -703,6 +711,34 @@ internal class TypedDataTest {
                 message = "{\"$includedType\": 1}",
             )
         }
+
+        @Test
+        fun `domain v1 object with integer revision`() {
+            val resolvedRevision = TypedData(
+                types = mapOf(
+                    domainTypeV1,
+                ),
+                primaryType = "StarknetDomain",
+                domain = domainObjectV1,
+                message = "{\"StarknetDomain\": 1}",
+            ).domain.resolvedRevision
+
+            assertEquals(Revision.V1, resolvedRevision)
+        }
```

Comment:
This case is already covered by all the other tests that are dealing with regular revision 1 typed data.
```suggestion
```